radlibDemoApp/www/index.html

```diff
@@ -1,11 +1,12 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <meta charset="utf-8" />
+      <meta charset="utf-8" />
       <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
 
         <link rel="stylesheet" type="text/css" href="css/index.css" />
         <link rel="stylesheet" type="text/css" href="css/topcoat-mobile-light.css" />
+        <link rel="icon" type="image/png" href="img/logosquare.png" />
```

Comment:
Nope, this is the favicon because I'm tired of looking at a blank white favicon when I'm debugging in the browser.